### ISSUE TITLE
multiples sessions

### ISSUE THREAD

**Issue by epm1989** [Forcepoint/fp-NGFW-SMC-python#56]
Hi friends, 
does the repo support multiple session instances?, either pointing to the same/different SMC  Server URL with different APIKEY - APIversion.
thanks in advance.

**Comment by amol-suruse** [comment]
Yes, it supports multiple session instances by pointing to the same or different SMC Server URL with the same or different API key and API version.

**Comment by amol-suruse** [comment]
you can ping me if require additional information.
Thanks
